samples/subsys/fs/nffs/README.rst

```diff
@@ -0,0 +1,35 @@
+.. _nrf52_NFFS_sample:
+
+nrf52 NFFS Read/Write Sample
+############################
+
+This is a simple application demonstrating how to read & write persistence data on 
```

Comment:
change ``persistence`` (noun) to ``persistent`` (adjective), and add in some missing articles ("the"):

    This is a simple application demonstrating how to read and write persistent
    data on the flash of the nRF52 SoC, using the NFFS file system.